src/Testcontainers/Builders/TestcontainersBuilderCosmosDbExtension.cs

```diff
@@ -0,0 +1,29 @@
+namespace DotNet.Testcontainers.Builders
+{
+  using System.Linq;
+  using DotNet.Testcontainers.Configurations;
+  using DotNet.Testcontainers.Containers;
+  using JetBrains.Annotations;
+
+  [PublicAPI]
+  public static class TestcontainersBuilderCosmosDbExtension
+  {
+    public static ITestcontainersBuilder<CosmosDbTestcontainer> WithCosmosDb(
+      this ITestcontainersBuilder<CosmosDbTestcontainer> builder, CosmosDbTestcontainerConfiguration configuration)
+    {
+      builder = configuration.Environments.Aggregate(builder, (current, environment)
+        => current.WithEnvironment(environment.Key, environment.Value));
+
+      return builder.WithImage(configuration.Image)
+        .WithPortBinding(configuration.DefaultPort, true)
```

Comment:
If you would like to keep the extension, change this to: `WithPortBinding(configuration.Port, configuration.DefaultPort)`.

Reply:
Removed this file.